client/pages/verify.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable no-console */
+// Module imports
+import React from 'react'
+// import PropTypes from 'prop-types'
+
+// Component imports
+import { actions, connect, actionStatus } from '../store'
+import { Router, Link } from '../routes'
+import PageWrapper from '../components/PageWrapper'
+import httpStatus from '../helpers/httpStatus'
+import PasswordField from '../components/PasswordField'
+import { passwordPattern } from '../data/RegExpr'
+
+
+// Component Constants
+
+@connect
+class Verify extends React.Component {
+  /***************************************************************************\
+    Class Properties
+  \***************************************************************************/
+
+  state = {
+    password: '',
+    submitted: false,
+    submitting: false,
+    validating: false,
```

Comment:
Whoops missed this one. Validating state can be removed here. it's never changed and validation is handled before the page is mounted.